doc/Windows.md

```diff
@@ -0,0 +1,28 @@
+## Local development with Windows in WSL 2 environment
+First you have to make sure, you are running Docker desktop in windows. See [Docker Desktop](https://www.docker.com/products/docker-desktop/) for more information.
```

Comment:
Did you try podman - there is a Windows binary ... https://github.com/containers/podman/releases/tag/v4.4.4

Reply:
windows -> Windows